recipes/default.rb

```diff
@@ -0,0 +1,85 @@
+#
+# Cookbook Name:: luigi
+# Recipe:: default
+#
+user node['luigi']['user'] do
+  system  true
+  shell   '/bin/false'
+  only_if { node['luigi']['setup_user'] }
+end
+
+group node['luigi']['group'] do
```

Comment:
I think I've already said that. But again... you need to associate luigi user to luigi group.